### PR TITLE
Some changes on strings PT-BR

### PR DESCRIPTION
**Pull request by bitcoinuser**

### CODE REVIEW

**Comment by bitcoinuser** [comment]
Hi @dpad85, could you please approve these changes and release a new version as soon as  possible? thanks.

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 1]
```diff
@@ -76,7 +76,7 @@
   <!-- //////////////// receive //////////////// -->
 
   <string name="receive_edit_title">Editar cobrança</string>
-  <string name="receive_edit_subtitle">Você pode alterar o valor solicitado e a descrição da cobrança.</string>
+  <string name="receive_edit_subtitle">Você pode alterar o valor e a descrição do pagamento.</string>
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:79]:
```suggestion
  <string name="receive_edit_subtitle">Você pode alterar o valor e a descrição.</string>
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 2]
```diff
@@ -99,7 +99,7 @@
   <string name="receive_swap_in_disclaimer_message"><![CDATA[
   Um endereço Bitcoin padrão será exibido a seguir.
   <br /><br />
-  Os bitcoins enviados para este endereço aparecerão na sua carteira após uma confirmação da rede.
+  Os bitcoins enviados para este endereço aparecerão na sua carteira após a primeira confirmação da rede.
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:102]:
```suggestion
  Os bitcoins enviados para este endereço aparecerão na sua carteira após 1 confirmação da rede.
```

File: app/src/main/res/values-pt-rBR/strings.xml [hunk 3]
```diff
@@ -99,7 +99,7 @@
   <string name="receive_swap_in_disclaimer_message"><![CDATA[
   Um endereço Bitcoin padrão será exibido a seguir.
   <br /><br />
-  Os bitcoins enviados para este endereço aparecerão na sua carteira após uma confirmação da rede.
+  Os bitcoins enviados para este endereço aparecerão na sua carteira após 1 confirmação da rede.
```
Comment (bitcoinuser) [inline app/src/main/res/values-pt-rBR/strings.xml:102]:
```suggestion
  Os bitcoins enviados para este endereço aparecerão na sua carteira após a 1ª confirmação da rede.
```

**Comment by fiatjaf** [comment]
Good improvements.

**Pull request merged**